### PR TITLE
Allow filtering by tags

### PR DESCRIPTION
**Pull request by jorge-wonolo**
Simplified approach to pass through tags as command line variables.

Based on https://github.com/skroutz/rspecq/pull/47 not moving forward I think this is a simpler approach to pass in specific rspec tag arg

### CODE REVIEW

**Review by jfragoulis — APPROVED** [review]
Thank you for the contribution. This looks great.

**Comment by jfragoulis** [comment]
On a side note, I really do not like commits without commit messages. Please, put it the effort to explain the why and implementation details if applicable, no matter how obvious it may seem.

**Comment by jfragoulis** [comment]
Squash the commits into one please.

**Comment by jorge-wonolo** [comment]
@jfragoulis thanks for the review, all your points should be addressed.

File: bin/rspecq [hunk 1]
```diff
@@ -12,7 +12,9 @@ def env_set?(var)
   ["1", "true"].include?(ENV[var])
 end
 
-opts = {}
+opts = {
+  :tags => []
```
Comment (jfragoulis) [inline bin/rspecq:16]: Sorry, I messed that, `tags: []`

**Review by jfragoulis — COMMENTED** [review]
`The rspec tag option allows **allows** ...`

File: bin/rspecq (hunk 1 above)
Comment (jorge-wonolo) [reply bin/rspecq]: OK fixed.

**Pull request merged**